MojioSDK/Client/AuthClient.swift

```diff
@@ -306,6 +306,10 @@ public class AuthClient: NSObject, AuthControllerDelegate {
                 failure(response: response.result.value as? NSDictionary)
             }
         }
```

Comment:
Print statements are useful for debugging and are not part of the release build #if/def'd